src/feed-list.scss

```diff
@@ -0,0 +1,120 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.75rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: $fd-feed-list-margin-end;
+    margin-right: $fd-feed-list-margin-start;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-start;
+      margin-right: $fd-feed-list-margin-end;
+    }
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem;
+
+    & + .#{$block}__body {
+      border-top: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+    }
+  }
+
+  &--no-border {
+    .#{$block}__body {
+      border-top: none;
+    }
+  }
+
+  &__link {
+    &--more {
+      text-transform: uppercase;
+    }
+  }
+
+  &__actions {
+    @include fd-reset();
+
+    float: right;
+    margin: 0.5rem;
+    position: relative;
+    top: -1rem;
+    right: -1rem;
+
+    @include fd-rtl() {
+      float: left;
+      right: 1rem;
+      left: -1rem;
+    }
+  }
+
+  &__actions--s {
+    padding-left: 0.75rem;
+    padding-bottom: 0.25rem;
+  }
+
+  &--group {
+    flex-direction: column;
+  }
+
+  &--link {
+    @include fd-fiori-focus();
+
+    cursor: pointer;
+  }
+
+  &__name {
+    @include fd-reset();
+
+    font-weight: bold;
+  }
+
+  &__text {
+    @include fd_reset();
+  }
+
+  &__footer {
+    @include fd-reset();
+
+    display: flex;
+    padding-top: 0.375rem;
+
+    &--byline {
+      color: var(--sapNeutralTextColor);
+
+      &:not(:last-child)::after {
+        content: '\2022';
+        padding: 0 0.25rem;
+      }
+    }
+  }
+
+  &--s {
+    .#{$block}__body {
+      display: block;
+    }
+
+    .#{$block}__text {
+      padding-top: 0.625rem;
+    }
+  }

```

Comment:
There should be something more for buttons
![image](https://user-images.githubusercontent.com/26483208/96567594-40ce8400-12c7-11eb-941f-9378f6bec8ed.png)

![image](https://user-images.githubusercontent.com/26483208/96567585-3d3afd00-12c7-11eb-8c90-f6ca00167528.png)